src/west/main.py

```diff
@@ -427,9 +429,10 @@ def set_zephyr_base(args):
                 # zephyr-env.sh/cmd was run in some other zephyr installation,
                 # and the user forgot about that.
                 log.wrn('ZEPHYR_BASE={}'.format(zb_env),
-                        'in the calling environment will be used, but was set '
-                        'to', zb_config, 'in west config.\n To disable this '
-                        'warning, execute '
+                        'in the calling environment will be used,\nbut was set'
+                        ' to "{}" subdirectory in [west config of]'
```

Comment:
I like this change and think it makes things easier to debug. Just for clarity, maybe instead of rel_zb_config, we can use its absolute path?

```
but was set to /home/john/somewhere-else/zephyr
in the configuration for the installation at /home/john/current-west-topdir
```

Reply:
You mean this?
> but was set to /home/john/current-west-topdir/zephyr
> in the configuration for the installation at /home/john/current-west-topdir

It's more explicit but I considered it a bit repetitive and I like printing the exact string that can be found in the config file.

Here's one of the other options I considered:
>  but was set to /home/john/west2//"zephyr" in west config.

It's compact and can be copied and re-used.

Reply:
I'm just struggling with this line in particular:

```
  "zephyr" subdirectory in [west config of] /home/john/west2
```

I think the square brackets and quotation marks are tripping me up, looks-wise.

How about:

```
but the zephyr.base config option in /home/john/west2 is "zephyr",
which asks for ZEPHYR_BASE=/home/john/west2/zephyr
```

or so?